pkgs/development/libraries/tracker/default.nix

```
@@ -103,7 +106,7 @@ stdenv.mkDerivation rec {
     # though, so we need to replace the absolute path with a local one during build.
     # We are using a symlink that will be overridden during installation.
     mkdir -p $out/lib
-    ln -s $PWD/src/libtracker-sparql/libtracker-sparql-3.0.so $out/lib/libtracker-sparql-3.0.so.0
+    ln -s $PWD/src/libtracker-sparql/libtracker-sparql-3.0${if stdenv.isDarwin then ".0.dylib" else ".so"} $out/lib/libtracker-sparql-3.0${if stdenv.isDarwin then ".0.dylib" else ".so.0"}
```

```suggestion
    ln -s src/libtracker-sparql/libtracker-sparql-3.0${lib.optionalString stdenv.isDarwin ".0"}${stdenv.hostPlatform.extensions.sharedLibrary} $out/lib/libtracker-sparql-3.0${if stdenv.isDarwin then ".0${stdenv.hostPlatform.extensions.sharedLibrary}" else "${stdenv.hostPlatform.extensions.sharedLibrary}.0"}
```
This can probably shortened a bit but I hope my idea gets transported

---

Done...?